<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>2packer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>9</y>
      <width>240</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="packTab">
     <attribute name="title">
      <string>Сжать</string>
     </attribute>
     <widget class="QGroupBox" name="selectFile2Pac">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Выберите файл</string>
      </property>
      <widget class="QLineEdit" name="selectedFile2PacLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="selectFile2PacButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="toPackFileButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сжать файл</string>
      </property>
     </widget>
     <widget class="QLabel" name="packFactorLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="packTimeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="unpackTab">
     <attribute name="title">
      <string>Расжать</string>
     </attribute>
     <widget class="QGroupBox" name="selectFile2Unpack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Выберите файл</string>
      </property>
      <widget class="QLineEdit" name="selectedFile2UnpackLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="selectFile2UnpackButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="toUnpackFileButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Расжать файл</string>
      </property>
     </widget>
     <widget class="QLabel" name="unpackTimeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
